### PR TITLE
Bugfix. False-positive LOCAL_VARIABLE_EARLY_DECLARATION

### LINKED ISSUE #488: False-positive LOCAL_VARIABLE_EARLY_DECLARATION
## Describe the bug
```kotlin
    fun `method name incorrect, part 4`() {
        val code = """
                  class TestPackageName {
                    fun methODTREE(): String {
                    }
                  }
                """.trimIndent()
        lintMethod(code, LintError(2, 7, ruleId, "${FUNCTION_NAME_INCORRECT_CASE.warnText()} methODTREE", true))
    }
```

## Expected behavior
Variable `code` is declared right before it's first used, so there should be no warning.

## Observed behavior
```
Warn (2:2) [LOCAL_VARIABLE_EARLY_DECLARATION] local variables should be declared close to the line where they are first used: <code> is declared on line <2> and is used for the first time on line <8>
```

## Environment information
* diktat version: 0.1.3

### PR DESCRIPTION
**Pull request by aktsay6**
### What's done:
  * Fixed bugs

This pull request closes #488

### CODE REVIEW

**Comment by aktsay6** [comment]
Can't reproduce this bug, so added test, that checks it

File: diktat-rules/src/test/resources/test/funcTest/FunctionalTestFile.kt [hunk 1]
```diff
@@ -3,3 +3,13 @@ package test.funcTest
 class FunctionalTestFile {
     val AAAA = 5
 }
+
+fun `method name incorrect, part 4`() {
```
Comment (petertrr) [inline diktat-rules/src/test/resources/test/funcTest/FunctionalTestFile.kt:7]: Please remove it from TestFile, you should add it into smoke test to test this case on the whole rule set provdier.

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LocalVariablesWarnTest.kt [hunk 2]
```diff
@@ -549,4 +549,19 @@ class LocalVariablesWarnTest : LintTestBase(::LocalVariablesRule) {
...
     }
+
+    @Test
+    @Tag(WarningNames.LOCAL_VARIABLE_EARLY_DECLARATION)
+    fun `should not trigger on triple quoted strings`() {
+        lintMethod(
+                """
+                    |class Example {
+                    |    fun some() {
+                    |       val code = ${"\"\"\"  class Some {\n \t\t fun for() {} \n \t} \"\"\".trimIndent()"}
```
Comment (petertrr) [inline diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LocalVariablesWarnTest.kt:560]: Why do you need `${}` here? Also, this is not exactly example from the issue, there are some different newlines there.

**Review by petertrr — CHANGES_REQUESTED** [review]

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LocalVariablesWarnTest.kt (hunk 2 above)
Comment (aktsay6) [reply diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LocalVariablesWarnTest.kt]: It is used to insert triple quotes

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LocalVariablesWarnTest.kt [hunk 3]
```diff
@@ -549,4 +549,24 @@ class LocalVariablesWarnTest : LintTestBase(::LocalVariablesRule) {
...
+    @Tag(WarningNames.LOCAL_VARIABLE_EARLY_DECLARATION)
+    fun `should not trigger on triple quoted strings`() {
+        val spacesForClassIndent = " ".repeat(18)
+        val spacesForFunIndent = " ".repeat(20)
+        val spacesForEndIndent = " ".repeat(16)
+        lintMethod(
+                """
+                    |class Example {
+                    |    fun some() {
+                    |       val code = ${"\"\"\"  \n${spacesForClassIndent}class Some {" +
```
Comment (petertrr) [inline diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LocalVariablesWarnTest.kt:563]: Why do you need explicit `\n` and spaces here? Why not write everything inside a string template? You can just surround triple quotes in curly braces, if that's why you did it.

**Review by petertrr — CHANGES_REQUESTED** [review]
So, after all, the issue isn't reproducible any more? Please write about it in the issue discussion (to not forget later).

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LocalVariablesWarnTest.kt (hunk 3 above)
Comment (aktsay6) [reply diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LocalVariablesWarnTest.kt]: Yeah, that seems better

**Review by petertrr — APPROVED** [review]

**Pull request merged**